Add missing param to client method

fractalwrench opened the pull request:
Addresses #211

Cawllec commented:
Is there a possibility that `getCurrentActivity` will return `null`?  Is this handled?

fractalwrench commented:
@Cawllec good catch - there's a chance that the activity could be null, I've updated to handle this case. I've also updated the iOS example app bundle id so that the app can be built
